### PR TITLE
overrides: add afterburn-4.3.2-1.fc31

### PR DESCRIPTION
**Pull request by jlebon**
We're trying to get FCOS working in Exoscale, and this is the final
missing piece feature wise.

### CODE REVIEW

File: manifest-lock.overrides.x86_64.yaml [hunk 1]
```diff
@@ -1,23 +1,19 @@
...
   # and the maintainer is not comfortable sending it to F31 yet.
   # https://src.fedoraproject.org/rpms/crypto-policies/pull-request/6#comment-35958
   crypto-policies:
     evra: 20191128-5.gitcd267a5.fc32.noarch
-  # New release of podman with fix for https://github.com/containers/libpod/issues/5306
-  # https://bodhi.fedoraproject.org/updates/FEDORA-2020-fc91673e24
-  podman:
-    evra: 2:1.8.1-2.fc31.x86_64
-  podman-plugins:
-    evra: 2:1.8.1-2.fc31.x86_64
```
Comment (dustymabe) [inline manifest-lock.overrides.x86_64.yaml:13]: I don't think we can remove this yet. Looking at bodhi this version was obsoleted by https://bodhi.fedoraproject.org/updates/FEDORA-2020-96d8964add which is not yet stable

Comment (dustymabe) [reply manifest-lock.overrides.x86_64.yaml:13]: if you check the stable repos and verify that we get this version or a newer version then we can remove it. If not then I think we still need it.

Comment (dustymabe) [reply manifest-lock.overrides.x86_64.yaml:13]: oops I was wrong. I was looking at podman-1.8.2-1.fc31

**Review by dustymabe — APPROVED** [review]

**Comment by dustymabe** [comment]
let's merge https://github.com/coreos/fedora-coreos-config/pull/323 first and rebase this one if there is a merge conflict

**Comment by jlebon** [comment]
No conflict! Merging!

**Pull request merged**